src/browsers/tob/shared/components/DotplotHeatmap.js

```diff
@@ -0,0 +1,241 @@
+              <g
+                key={tick}
+                transform={`translate(0, ${yScaleLocal(tick) + yScaleLocal.bandwidth() / 2})`}
+              >
+                <text key={tick} style={{ textAnchor: 'end', alignmentBaseline: 'middle' }} x={-6}>
+                  {tick}
+                </text>
+                <line x2={-3} stroke="black" />
+                <line x2={`${innerWidth}`} stroke="lightgrey" />
+              </g>
+            ))}
+          </g>
+          <g id="tiles">
+            {data.length ? (
+              data.map((item) => (
+                <circle
+                  key={`${accessors.x(item)},${accessors.y(item)}`}
+                  r={sizeScaleLocal(accessors.size(item))}
+                  cx={xScaleLocal(accessors.x(item)) + xScaleLocal.bandwidth() / 2}
+                  cy={yScaleLocal(accessors.y(item)) + yScaleLocal.bandwidth() / 2}
+                  stroke="black"
+                  fill={`${colorScaleLocal(accessors.color(item))}`}
+                  onClick={onClick}
+                />
+              ))
+            ) : (
+              <g transform={`translate(${innerWidth / 2}, ${innerHeight / 2})`}>
+                <text
+                  style={{ textAnchor: 'middle', alignmentBaseline: 'middle', fontSize: 'larger' }}
+                >
+                  No data to display.
+                </text>
+              </g>
+            )}
+          </g>
+          {data.length && (
+            <g
+              id="sizeLegend"
+              transform={`translate(${
+                width - margin.right - margin.left + sizeScaleLocal.range()[1] + 5
+              }, 20)`}
+            >
+              <text fontWeight="bold"> P value </text> <br />
+              {sizeScaleLocal
+                .ticks(6)
+                .filter((size) => size < 1)
+                .map((tick, i) => (
+                  <g
+                    key={`sizeScaleLocal ${tick}`}
+                    transform={`translate(0, ${40 + sizeScaleLocal.range()[1] * 2 * i})`}
+                  >
+                    <circle r={sizeScaleLocal(tick)} stroke="black" fill="none" />
+                    <text x={sizeScaleLocal.range()[1] + 5} style={{ alignmentBaseline: 'middle' }}>
+                      {tick.toFixed(1)}
+                    </text>
+                  </g>
+                ))}
+            </g>
+          )}
+          {data.length && (
+            <g
+              id="colourLegend"
+              transform={`translate(${
+                width - margin.right - margin.left + sizeScaleLocal.range()[1] + 100
+              }, 20)`}
+            >
+              <text fontWeight="bold"> Expression </text> <br />
+              <rect
+                transform="translate(0, 15)"
+                height="240"
+                width="40"
+                fill="url(#linear-gradient)"
+                stroke="black"
+              />
+              {colorScaleLocal.ticks(6).map((tick, i) => (
```

Comment:
For the same reasons as with the previous legend, change ticks to 2 for the min and max of the scale.